src/envs/behavior.py

```diff
@@ -433,8 +443,8 @@ def _create_type_combo_name(original_name: str,
 
 
 def make_behavior_option(name: str, types: Sequence[Type], params_space: Box,
-                         env: "behavior_env.BehaviorEnv",
-                         controller_fn: Callable,
+                         env: "behavior_env.BehaviorEnv", planner_fn: Callable,
+                         policy_fn: Callable, option_model_fn: Callable,
```

Comment:
+1. Actually it would be really good to get rid of these generic `Callable` types altogether... running `git grep "Callable" | grep -v "Callable\[" | grep -v "from typing"` gives me:
```
src/envs/behavior.py:                         controller_fn: Callable,
src/envs/behavior.py:                         object_to_ig_object: Callable,
```
on master right now. Ideally, all the Callables should have clearly specified parameter and return types. If that's not able to be done in this case, perhaps you need a base class with a well-defined API for what a policy_fn and option_model_fn should do.

Reply:
I updated this and got rid of the `type: ignore` comments, but there's a weird mypy error now that I can't figure out (perhaps this is because of the way we call `make_behavior_options` by iterating through the elements of a list?)